app/views/admin/accounting/transactions/_transactions_grid_definition.html.slim

```diff
@@ -10,3 +10,20 @@
   - g.column name: t('activerecord.attributes.accounting.transaction.amount'),
     filter: false do |transaction|
     - format_currency transaction.amount, transaction.currency
+  - g.column name: t('activerecord.attributes.accounting.transaction.interest_paid'),
+    filter: false do |transaction|
+    - if transaction.loan_transaction_type == 'repayment'
+      - format_currency transaction.change_in_interest, transaction.currency
```

Comment:
I think this will show the opposite value of what you want, because the `change_in_interest` is going to be negative for repayments. I would consider adding methods for `interest_paid` and `capital_returned` that return `-change_in_interest` and `-change_in_principal` respectively.